ios/RNAccountKitViewController.m

```diff
@@ -49,11 +49,11 @@ - (void)loginWithPhone: (RCTPromiseResolveBlock)resolve
 {
     _resolve = resolve;
     _reject = reject;
+    NSString *prefillPhone - self.initialPhone;
```

Comment:
@gaguirre Yep! I fixed this locally thought I pushed this change! I will work on this over the weekend and do a test on android too! Thanks for pointing this out!